scripts/sql/xavier-changes-16-03-2016/updates.sql

```diff
@@ -0,0 +1,72 @@
+﻿set search_path = aatams, public;
+
+--------------------
+---- Inadvertent entry into the production db, needs to be deleted.
+DELETE FROM sensor where transmitter_id = 'A69-1303-62731' AND transmitter_type_id = 34;
+
+
+
+
+--------------------
+---- VR2-AR device model inconsistency. In the device_model table the VR2AR receiver model is stored both as 'VR2-AR' and VR2AR', it should only be stored as 'VR2AR'.
+UPDATE device SET model_id = 129558776 WHERE model_id = 117465295 -- Transforms all receivers registered as VR2-AR into VR2AR
+DELETE FROM device_model WHERE model_name = 'VR2-AR'; -- Delete wrongly named receiver model
+
+---- Test results
+-- SELECT DISTINCT receiver_name FROM detection WHERE receiver_name LIKE 'VR2-AR%' -- Returns 0 row
+-- SELECT DISTINCT receiver_name FROM detection WHERE receiver_name LIKE 'VR2AR%' -- Returns 10 rows
+-- SELECT COUNT(*) FROM device WHERE model_id = 117465295 -- VR2-AR -- Count = 20
+-- SELECT COUNT(*) FROM device WHERE model_id = 129558776 -- VR2AR -- Count = 0
+
+
+
+
+--------------------
+---- Deletion of receiver for JCU student 
+DELETE FROM receiver_deployment 
+USING device
+WHERE device.id = receiver_deployment.receiver_id 
+AND device.serial_number = '102044' AND initialisationdatetime_timestamp IS NULL;
+
+
+
+
+--------------------
+---- Add 'RANGE TEST' tag as a new type of transmitter in the transmitter_type table. TO DO: update pinger tags which we know are range test tags in the sensor table (Andre to provide list).
+INSERT INTO transmitter_type (id, version, transmitter_type_name) VALUES
+(37, 0, 'RANGE TEST');
+
```

Comment:
Its possible to do this through the UI (show sensor and click on an existing transmitter_type or go directly to aatams/transmitterType/list).  We should use the UI in preference to using sql.